templates/phenotype/EpilepsyClassification2017.json

```diff
@@ -0,0 +1,394 @@
+{
+    "EpilepsyClassification":{
+        "Description": "ILAE classification of the epilepsies",
+        "TermURL": "",
+        "Citation": "Scheffer, I. E., Berkovic, S., Capovilla, G., Connolly, M. B., French, J., Guilhoto, L., ... & Nordli, D. R. (2017). ILAE classification of the epilepsies: position paper of the ILAE Commission for Classification and Terminology. Epilepsia, 58(4), 512-521."
+    },
+    "SeizureType": {
+        "LongName": "Seizure type",
+        "Description": "Seizures are classified by onset: Generalised, Focal and Unknown",
+        "Levels": {
+            "Generalised": {
+                "Motor": [
+                    "Generalized tonic-clonic and variants",
```

Comment:
Should we leave out spaces and dashes here?

Reply:
perhaps replacing spaces with `_` would result in better readability? dashes could just remain?

Reply:
thank you @dorahermes and @sappelhoff !
 
In the .tsv file example, I have used  '/' to separate levels (flattening) and camelCase instead of spaces or punctuation.

JSON schema left as is so it would be readable.

Reply:
+1 on replacing spaces with _

Reply:
The CamelCase seems to work quite well in the .tsv. Implementing this in the json file will work well I think.